src/plays/personal-profile-card/local-storage.ts

```diff
@@ -0,0 +1,27 @@
+const set = (key: string, value: object) => {
```

Comment:
we already have useLocalStorage hook in the common/hooks foler. can you make use of it?

Reply:
Used useLocalStorage hook instead my own utility function.